test/unit_test/deepdoc/parser/test_opendataloader_parser.py

```
@@ -182,6 +194,25 @@ def test_json_doc_response_returns_sections(self, tmp_path):
+        docling = {
+            "texts": [
+                {"label": "paragraph", "text": "Hello from Docling", "prov": [{"page_no": 1, "bbox": {"l": 0, "t": 0, "r": 100, "b": 20, "coord_origin": "TOPLEFT"}}]},
+                {"label": "section_header", "text": "A Title", "prov": [{"page_no": 1, "bbox": {"l": 0, "t": 20, "r": 200, "b": 40, "coord_origin": "TOPLEFT"}}]},
+            ],
+            "tables": [],
+            "pictures": [],
+        }
+        resp = self._mock_response(docling=docling)
+
+        with mock.patch.object(p, "__images__"), mock.patch("requests.post", return_value=resp):
+            sections, tables = p.parse_pdf(filepath="doc.pdf", binary=b"%PDF", parse_method="pipeline")
```

Ruff (0.15.21)

[warning] Unpacked variable `tables` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)